src/utils.js

```diff
@@ -12,6 +12,10 @@ function padlz (n, len) {
   return n;
 }
 
+function configureFromEnv () {
```

Comment:
maybe something lithe this instead ?

```suggestion
function isTTYMode () {
```